src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -68,4 +68,9 @@ def handle_exception(e):
                 formatted_exception.status_code,
             )
 
+        @app.before_request
+        def log_request_info():
+            app.logger.info("Headers: %s", request.headers)
+            app.logger.info("Body: %s", request.get_data())
```

Comment:
Headers and Body should be debug level log. 
For INFO we might have something like URL, STATUS Code, Duration, And error response /stack trace if any error.

Reply:
duplicated